jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/ContentDataImpl.java

```diff
@@ -51,6 +68,53 @@ public void setType(String type) {
 	public void setContent(byte[] content) {
 		this.content = content;
 	}
+	
+	@Override
+	public Object getContentObject() {
+        MarshallerReaderContext context = null;
+        try {
+            ByteArrayInputStream stream = new ByteArrayInputStream(content);
```

Comment:
ah, in that case return null will make more sense. This is too limited without extra information like class loader or marshalling strategies.